Api/test/userTest.js

```diff
@@ -0,0 +1,61 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import {
+  res
+} from "express";
+const expect = chai.expect;
+
+//Assertion
+chai.should();
+chai.use(chaiHttp);
+
+// To do
+describe('User API', () => {
+  //Testing Different Routes 
+  // Get route
+  describe("Get /api/src/routes/userRoute", () => {
+    it("It should Get all users", (done) => {
+      chai.request(server)
+        .get("/api/src/routes/userRoute")
```

Comment:
```suggestion
        .get("/api/src/routes")
```

Reply:
Fixed, but the tests fails because of the payload.